commit 6060653e786c0597389c686f275a0c787a18d9bb
Author: Palmer Dabbelt <palmer@dabbelt.com>
Date:   Mon Feb 25 15:56:57 2013 -0800

    Fix GCC 4.6 build
    
    This was a suprious warning, but it was still worth looking at.

diff --git a/src/tek/processor/latex.c b/src/tek/processor/latex.c
index f5c75d5..20746d9 100644
--- a/src/tek/processor/latex.c
+++ b/src/tek/processor/latex.c
@@ -127,7 +127,6 @@ void process(struct processor *p_uncast, const char *filename_input,
     char *filename;
     char *phonydeps;
     char *biblio;
-    char *dir_of_file;
 
     phonydeps = NULL;
 
@@ -157,7 +156,7 @@ void process(struct processor *p_uncast, const char *filename_input,
     strcat(cache_dir, CACHE_DIR);
 
     /* The directory this file resides in */
-    dir_of_file = talloc_strndup(c, filename, basename_len(filename));
+    talloc_strndup(c, filename, basename_len(filename));
 
     /* The preprocessed file */
     pp_file = talloc_array(c, char,
